sshd/terminal.go

```diff
@@ -161,10 +187,27 @@ func (t *Terminal) listen(requests <-chan *ssh.Request) {
 				err := t.SetSize(width, height)
 				ok = err == nil
 			}
+		case "env":
+			var v EnvVar
+			if err := ssh.Unmarshal(req.Payload, &v); err != nil {
```

Comment:
```suggestion
			if err := ssh.Unmarshal(req.Payload, &v); err == nil {
```

Use env var if no error, this fix already applied in #333